src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -177,31 +178,28 @@ String serviceHeader() {
 
   @VisibleForTesting
   static String getGaxVersion() {
-    String gaxVersion = DEFAULT_VERSION;
-    Properties gaxProperties = new Properties();
-    try {
-      gaxProperties.load(
-          InstantiatingChannelProvider.class
-              .getResourceAsStream("/com/google/api/gax/gax.properties"));
-      gaxVersion = gaxProperties.getProperty("version");
-    } catch (IOException e) {
-      e.printStackTrace(System.err);
-    }
-    return gaxVersion;
+    String gaxVersion = loadGaxProperty("version");
+    return gaxVersion != null ? gaxVersion : DEFAULT_VERSION;
```

Comment:
I don't feel comfortable of returning `DEFAULT_VERSION` from loadGaxProperty since loadGaxProperty is a general helper that conceptually returns any property in the properties file for the given key.